.coveragerc

```diff
@@ -259,6 +259,7 @@ omit =
     homeassistant/components/ecobee/__init__.py
     homeassistant/components/ecobee/binary_sensor.py
     homeassistant/components/ecobee/climate.py
+    homeassistant/components/ecobee/fan.py
```

Comment:
Why is it added here when you have implemented tests?

Reply:
@gjohansson-ST  , the test does not provide 100% coverage, I did not find the way to test the [async_setup_entry](https://github.com/home-assistant/core/pull/83645/files#diff-a512bf42b37bf296aa4c7a4460458217d634f473106d25f58cb7dd0acc4b4553R33) so I added the file to `.coveragerc` (This is also inline with other Ecobee entities (climate, sensors, etc))

Let me know if there is a better way to do it.

Reply:
You need to mock a config entry and use that to setup the integration.
Search the codebase for `MockConfigEntry` to see multiple examples or here is one: https://github.com/home-assistant/core/blob/dev/tests/components/sensibo/conftest.py#L23

See if you can make it to not add it to coverage for exclusion.

Reply:
Looking at this, Thank a lot